universum/configuration_support.py

```diff
@@ -12,6 +15,203 @@
 
 skip_attributes = {"children", "critical", "skip_numbering_level"}
 
+ProjectConfigKeyType = Literal["name", "command", "environment", "artifacts", "report_artifacts",
```

Comment:
The benefits of using this annotation are not clear to me.
There is only one usage in the code with only one key name checked.
My guess is that you are going to expand the usage in further updates.
Please confirm it.